src/main/java/net/snowflake/client/core/HttpUtil.java

```diff
@@ -69,6 +65,8 @@
   static final int DEFAULT_IDLE_CONNECTION_TIMEOUT = 5; // secs
   static final int DEFAULT_DOWNLOADED_CONDITION_TIMEOUT = 3600; // secs
 
+  public static final String JDBC_TTL_JVM = "net.snowflake.jdbc.ttl";
```

Comment:
Change the name to JDBC_TTL? JVM means Java virtual machine so I don't see anything related to that.

Reply:
@sfc-gh-azhan , it was just named that because it's a JVM parameter instead of a connection parameter. Took out the JVM part of the name!

Reply:
It is not a JVM parameter though https://www.baeldung.com/jvm-parameters

Reply:
Ah really? Maybe my vocab is wrong. I guess it could just be called a command line parameter, idk. Can be set in command line with `-Dnet.snowflake.jdbc.ttl=<value>`

Reply:
I guess it counts as a kind of JVM parameter, or say custom JVM property

Reply:
I think the suffix `_JVM` is fine. The intention is this parameter is for the JVM level and not connection level.

Reply:
Maybe my context is a little different. I work on Java UDF recently and when we start the UDF we need to launch JVM to run Java code and we treat JVM parameters like -xmx, -xms, etc. Another example is `networkaddress.cache.ttl` in https://docs.oracle.com/javase/7/docs/technotes/guides/net/properties.html. This parameter, I mean, from the naming, is more like a Snowflake JDBC parameter though. BTW, it also better to specify what ttl it is if we want to expose it, e.g., `net.snowflake.jdbc.http.ttl`.